pkg/planner/core/expression_rewriter.go

```diff
@@ -1432,8 +1442,12 @@ func (er *expressionRewriter) Leave(originInNode ast.Node) (retNode ast.Node, ok
 	}
 
 	switch v := inNode.(type) {
-	case *ast.AggregateFuncExpr, *ast.ColumnNameExpr, *ast.ParenthesesExpr, *ast.WhenClause,
-		*ast.SubqueryExpr, *ast.ExistsSubqueryExpr, *ast.CompareSubqueryExpr, *ast.ValuesExpr, *ast.WindowFuncExpr, *ast.TableNameExpr:
+	case *ast.AggregateFuncExpr:
+		if v.F == ast.AggFuncCount {
+			er.collectPrivsForCount()
```

Comment:
To collect any columns' `SELECT` privilege of the table(s)

Reply:
@CbcWestwolf Don't other aggregation operations require privilege checks?